### ISSUE TITLE
[Slider] Windows Phone (Edge) - Cannot slide, can only tap to get to value

### ISSUE THREAD

**Issue by doublesidedstickytape** [zurb/foundation-sites#10097]
#### How to reproduce this bug:

1. I'm using a Lumia 550 and a Lumia 950 (latest build/update of Windows 10 Mobile)
2. Open Microsoft Edge
3. Navigate to the Slider Docs (http://foundation.zurb.com/sites/docs/slider.html)

#### What should happen:
Should be able to touch the square and slide between left and right

#### What happened instead:
In Edge, it does not respond. I can only tap a blank area of the bar for it to move.

#### Browser(s) and Device(s) tested on:
 I'm using a Lumia 550 and a Lumia 950 (latest build/update of Windows 10 Mobile)

#### Test case link:
http://foundation.zurb.com/sites/docs/slider.html

**Comment by IamManchanda** [comment]
Yup i confirm i tested it ... seems like an issue ...
cc @Owlbertz

**Comment by Owlbertz** [comment]
@IamManchanda how were you able to reproduce? Any chance doing that via emulation? I do not own a Windows-Phone and using emulation in with Lumia 950 in Edge works fine as far as I can see.

**Comment by BernhardWebstudio** [comment]
I was able to reproduce the bug on my Lumia as well as on a Surface tablet, where the slider works using the mouse, but not using touch.

**Comment by kball** [comment]
@zurb/yetinauts anyone have windows touch device they can use to debug & track this down?

**Comment by BernhardWebstudio** [comment]
The bug seems to be related to the flags set in [about:flags](about:flags). After reseting them to default, the slider seems to work as it should on my devices. It is especially the flag _Enable touch events_ which has to be set to - somewhat counterintuitive - _always disabled_.

**Comment by DanielRuf** [comment]
> The bug seems to be related to the flags set in about:flags. After reseting them to default, the slider seems to work as it should on my devices

On which devices did this fix it?
Can this still be reproduced with the current Edge version?

**Comment by BernhardWebstudio** [comment]
It fixed the error on both Lumia and Surface in Edge. I just checked on the Surface, on which the bug does not occur anymore, even with the flag set to _enable_. Maybe OP @doublesidedstickytape has more information from his side and devices?

**Comment by DanielRuf** [comment]
Closing as we did not receive further feedback.
